tests/extend_dac/Makefile

```diff
@@ -0,0 +1,11 @@
+include ../Makefile.tests_common
+
+USEMODULE += extend_dac
+
+TEST_ON_CI_WHITELIST += all
+
+include ./Makefile.include
+include $(RIOTBASE)/Makefile.include
+
+test:
```

Comment:
Explicitly referencing the test is no longer necessary. If ```./tests/01-run.py``` exists, then it will be used.